### PR TITLE
[synthetics] Update monitor status rule docs

### LINKED ISSUE #4164: [Request]: Document how to move from the Uptime Rule to the Synthetics Rule
### Description

Uptime is deprecated and will be removed in early 2026 (as called out in the [release note](https://www.elastic.co/guide/en/kibana/current/release-notes-8.15.0.html#deprecations-8.15.0)).

As part of this, [improvements](https://github.com/elastic/kibana/pull/186585) are being made to the Synthetics rule type, to provide the capabilities currently available in the Uptime rule, but in the Synthetics rule.

It's worth noting that we have a [Support KB article](https://support.elastic.co/knowledge/053bcbc4) that even describes how users can use the Uptime rule in place of a more capable Synthetics rule.  This will no longer be necessary, and the preferred approach will be to use the Synthetics rule (it will be the only viable approach in Elastic v9).

This issue is to document how users who have an existing Uptime Rule they are using for their Synthetics monitors, can replace this with the upcoming enhances Synthetics rule.

### Resources

The new Synthetics rule capability is being added in https://github.com/elastic/kibana/pull/186585

### Which documentation set does this change impact?

Stateful and Serverless

### Feature differences

The feature is the same between stateful and serverless

### What release is this request related to?

After https://github.com/elastic/kibana/pull/186585 is merged

### Collaboration model

Other (please describe below)

### Point of contact.

It is expected that the documentation team and Management UX team will work together on the content

**Main contact:**
- @shahzad31 

**Stakeholders:**
- @drewpost 
- @jasonrhodes 
- @paulb-elastic

### LINKED ISSUE #4337: [Request]: Synthetics custom alerts
### Description

In 8.16 with this PR https://github.com/elastic/kibana/pull/186585
User can create custom status alerts. 

We need to document different conditions and threshold available in flyout and also filtering monitors.

### Resources

https://github.com/elastic/kibana/pull/186585

### Which documentation set does this change impact?

Stateful and Serverless

### Feature differences

it works same on both

### What release is this request related to?

8.16

### Collaboration model

The documentation team

### Point of contact.

**Main contact:** @shahzad31 @dominiqueclarke @justinkambic 

**Stakeholders:**

### PR DESCRIPTION
**Pull request by colleenmcginnis**
## Description

The Synthetics monitor status rule type was improved in https://github.com/elastic/kibana/pull/186585. This PR makes a few updates to the [Create a monitor status rule](https://observability-docs_bk_4345.docs-preview.app.elstc.co/guide/en/observability/master/monitor-status-alert.html) doc including:

* Marking the Uptime rule as deprecated and making it clear that the Synthetics rule is the right choice moving forward.
* Fully documenting the Synthetics monitor status rule type with details on the available filters, conditions, action types, and action variables.
* Documenting how users who have an existing Uptime rule that they are using for their Synthetics monitors can replace the Uptime rule with the Synthetics rule.
  * @shahzad31 @paulb-elastic  I'm not sure if this section ([preview](https://observability-docs_bk_4345.docs-preview.app.elstc.co/guide/en/observability/master/monitor-status-alert.html#migrate-monitor-rule)) is what you had in mind. I'm not sure there's a 1:1 comparison for everything included in the Uptime rule unless I'm misunderstanding. For example, I don't think you can recreate the Uptime rule's availability condition exactly using the new Synthetics rule conditions (would you use a [synthetics SLI](https://github.com/elastic/kibana/pull/177842) instead?). Let me know if I'm missing something or if there's a different way we should approach this.

Note: I documented the Uptime and Synthetics rules on the same page titled "Create a monitor status rule" because the "Alerts and rules" dropdown in both the Uptime app and Synthetics UI uses the language "Monitor status rule", but I could be persuaded to document them separately. Either way, starting in 9.0.0, I imagine the Uptime content on this page will go away.

### Documentation sets edited in this PR

_Check all that apply._

- [x] Stateful (`docs/en/observability/*`)
- [x] Serverless (`docs/en/serverless/*`)
- [ ] Integrations Developer Guide (`docs/en/integrations/*`)
- [ ] None of the above

### Related issue

Closes #4164
Closes #4337

## Checklist

- [x] Product/Engineering Review
- [x] Writer Review

### Follow-up tasks

_Select one._

* This PR does _not_ need to be ported to another doc set because:
  - [ ] The concepts in this PR only apply to one doc set (serverless _or_ stateful)
  - [x] The PR contains edits to both doc sets (serverless _and_ stateful)
* This PR needs to be ported to another doc set:
  - [ ] Port to stateful docs: \<link to PR or tracking issue>
  - [ ] Port to serverless docs: \<link to PR or tracking issue>

### CODE REVIEW

**Comment by paulb-elastic** [comment]
I've not had a chance to look through this fully, but a couple of comments:

> starting in 9.0.0, I imagine the Uptime content on this page will go away

Uptime is not being removed now in 9.0.0.  It's still deprecated, but won't be removed until a later date (the [release note](https://www.elastic.co/guide/en/kibana/current/release-notes-8.15.0.html#deprecations-8.15.0) was updated recently to reflect that change).

[This deprecated content](https://observability-docs_bk_4345.docs-preview.app.elstc.co/guide/en/observability/master/monitor-status-alert.html#monitor-status-alert-uptime): whilst it's true that Uptime is deprecated (and as such, the corresponding rule), is this section clear that _Uptime_ is deprecated and the suggestion is to use _Synthetics_ as a whole solution (specifically, is there a chance it could be misread and people think they should use the Synthetics rule if still using Uptime)?

**Comment by shahzad31** [comment]
In PR preview, Monitor status rule still leads me to Uptime monitor status rule docs , i will recommend replacing that link with synthetics status alert !!

image

File: docs/en/observability/monitor-status-alert.asciidoc [hunk 1]
```diff
@@ -1,14 +1,158 @@
...
+If you are using the Uptime monitor status rule with a monitor created with Elastic Synthetics, you should migrate the Uptime monitor status rule to the Synthetics monitor rule.
+Learn how in <<migrate-monitor-rule>>.
+====
+
 Within the {uptime-app}, create a **Monitor Status** rule to receive notifications
 based on errors and outages.
 
-. To access this page, go to **{observability}** -> **Uptime**.
-. At the top of the page, click **Alerts and rules** -> **Create rule**.
+. To access this page, go to **{observability}** → **Uptime**.
```
Comment (shahzad31) [inline docs/en/observability/monitor-status-alert.asciidoc:154]:
```suggestion
. To access this page, go to **{observability}** → **Synthetics**.
```

**Comment by shahzad31** [comment]
@colleenmcginnis we don't allow know creating availability alert from status rules, it doesn't make sense as far as i can understand. if you want you can mention Synthetics Availability SLI indicator, that's more suited for that kind of scenerio, where user want to get alerted on availability.  So user looking for availability alert can learn about Synthetics Availability SLI indicator.

**Comment by colleenmcginnis** [comment]
>In PR preview, Monitor status rule still leads me to Uptime monitor status rule docs , i will recommend replacing that link with synthetics status alert !!

You'll need to look at this link (`master` instead of `current`): https://observability-docs_bk_4345.docs-preview.app.elstc.co/guide/en/observability/master/monitor-status-alert.html#migrate-monitor-rule. (Then we can backport to publish to other branches.)

**Review by shahzad31 — DISMISSED** [review]
LGTM , other than if we want to leave a comment about availability condition being replaced with Synthetics Availability SLI !!

File: docs/en/observability/monitor-status-alert.asciidoc [hunk 2]
```diff
@@ -1,14 +1,158 @@
...
+
+[WARNING]
+====
+*The Uptime app and the Uptime monitor status rule are deprecated as of version 8.15.0.*
+
+If you are using the Uptime monitor status rule with the Uptime app, you should migrate the Uptime monitor and the Uptime monitor status rule to Elastic Synthetics and the Synthetics monitor rule.
+
+If you are using the Uptime monitor status rule with a monitor created with Elastic Synthetics, you should migrate the Uptime monitor status rule to the Synthetics monitor rule.
+Learn how in <<migrate-monitor-rule>>.
+====
```
Comment (bmorelli25) [inline docs/en/observability/monitor-status-alert.asciidoc:149]: Did you mean to duplicate this on this page?

File: docs/en/observability/monitor-status-alert.asciidoc [hunk 3]
```diff
@@ -80,9 +224,198 @@ image::images/monitor-status-alert-default-message.png[Default notification mess
...
+
+[discrete]
+==== Actions
+
+The default messages for the Uptime monitor status rule and Synthetics monitor status rule are different,
+but you can recreate similar messages using <<synthetic-monitor-action-variables,Synthetics monitor status rule action variables>>.
+
+// Not sure if this is actually helpful AND not sure if there are equivalents for
+// availability checks
+
```
Comment (bmorelli25) [inline docs/en/observability/monitor-status-alert.asciidoc:296]: @\reviewers it looks like there's still a question to be answered here

Comment (shahzad31) [reply docs/en/observability/monitor-status-alert.asciidoc:296]: i didn't understand the question @colleenmcginnis can you please elaborate?

Comment (colleenmcginnis) [reply docs/en/observability/monitor-status-alert.asciidoc:296]: This refers to the commented out table below that compares specific action variables between the Uptime monitor status rule and the Synthetics monitor status rule.

**Review by shahzad31 — DISMISSED** [review]
LGTM !!

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-3mqsu6qby-elastic-dev.vercel.app

* Built with commit ea68830d6b15c699411e41dbf01eed5e1c6754ee

> Issues? Visit #next-docs in Slack

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**